recipes/bioconda-repodata-patches/gen_patch_json.py

```diff
@@ -123,6 +123,13 @@ def _gen_new_index(repodata, subdir):
                     deps[i] = 'r-base >={},<{}'.format(minVersion, maxVersion)
                     break
 
+        # Bioconductor data packages are noarch: generic and incorrectly pin curl to >=7.38.1,<8, rather than >=7,<8
+        if subdir == "noarch" and record_name.startswith('bioconductor-') and has_dep(record, "curl"):
+            for i, dep in enumerate(deps):
+                if dep.startswith('curl >=7.'):
+                    deps[i] = 'curl >=7,<8.0a0'
```

Comment:
I think it should even not pin curl at all. We only use the command line interface, and there are no run_exports in curl. We can also be quite sure that the CLI options we use will never change.

Reply:
That's fair, I'll make that change. It should make the environment easier to solve.